kong/runloop/plugin_servers/init.lua

```diff
@@ -15,14 +15,25 @@ local kong = kong
 local ngx_var = ngx.var
 local ngx_sleep = ngx.sleep
 local worker_id = ngx.worker.id
+local req_start_time = ngx.req.start_time
```

Comment:
https://github.com/openresty/lua-nginx-module#ngxreqstart_time

This function is not available in the `init` phase, I think we can't localize them.

Reply:
I think you may misunderstand the meaning of `context`.
The context of ngx_lua api means it is available and should be called in these phases.

If we read the source of ngx_lua and lua-resty-core,
we will find that these apis have been loaded into lua land.